src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -113,6 +113,8 @@ public class SingleRun
             public bool ReadOnly;
             private readonly ConcurrentLruRegexCache _regexCache = new ConcurrentLruRegexCache(1024);
 
+            private static Dictionary<string, CultureInfo> _cultureInfoDict;
```

Comment:
Move this to `CultureHelper` and do this:

```
        private static readonly Lazy<Dictionary<string, CultureInfo>> CulturesLazy = new Lazy<Dictionary<string, CultureInfo>>(() => CultureInfo.GetCultures(CultureTypes.AllCultures).ToDictionary(x => x.Name, x => x, StringComparer.OrdinalIgnoreCase));

        internal static Dictionary<string, CultureInfo> Cultures => CulturesLazy.Value;
```

Reply:
done